hieradata/site/cp/role/nfsserver.yaml

```diff
@@ -4,6 +4,10 @@ classes:
   - "profile::core::nfsserver"
   - "profile::core::nfsclient"
 
+lookup_options:
```

Comment:
Nonblocker: If possible we should push this lookup to `manifests/site.pp`

Reply:
I agree, that definitely makes sense.